script/hassfest/__main__.py

```diff
@@ -117,14 +117,11 @@ def get_config() -> Config:
             "Generate is not allowed when limiting to specific integrations"
         )
 
-    if (
-        not parsed.integration_path
-        and not pathlib.Path("requirements_all.txt").is_file()
-    ):
+    if not parsed.integration_path and not Path("requirements_all.txt").is_file():
         raise RuntimeError("Run from Home Assistant root")
 
     return Config(
-        root=pathlib.Path(".").absolute(),
+        root=Path().absolute(),
```

Comment:
Why did we remove the period?

Reply:
Because no arguments and "." as argument is the same, https://docs.astral.sh/ruff/rules/path-constructor-current-directory/